app/src/pages/landing.js

```diff
@@ -1,9 +1,36 @@
 import React from "react";
+import { Link } from "react-router-dom";
 import ReactDOM from "react-dom";
+import landing_svg from '../img/landing.svg';
 
 const LandingPage = () => {
     return (
-        <h1>Landing Page</h1>
+        <div className = 'landing-page'>
+            <h1 className='logo'>CoderHub</h1>
+
+            <h1>Share, Interact,</h1>
```

Comment:
first of all, i think u gotta change the html structure inside this component
like three separate boxes; one for the title, one for the rest of the content and one last div for the img. this will help u style it with flex easier, at least in the way that should be displayed in acording to the design